### PR TITLE
Update from template

### PR DESCRIPTION
**Pull request by aaron-lane**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -84,11 +88,11 @@ test_unit: test_migrate test_preconditions
 
 # Integration tests
 .PHONY: test_integration
-test_integration: ## Run integration tests
```
Comment (morgante) [inline Makefile]: Why are we dropping these? Command explanations are useful.

File: test/fixtures/full/outputs.tf [hunk 2]
```diff
@@ -1 +0,0 @@
-../shared/outputs.tf
```
Comment (morgante) [inline test/fixtures/full/outputs.tf]: Please don't drop the symlinks.

File: test/make.sh [hunk 3]
```diff
@@ -14,83 +14,149 @@
...
+# against all directory paths which contain *.tf files.
 function check_terraform() {
+  set -e
   echo "Running terraform validate"
-  #shellcheck disable=SC2156
-  find . -name "*.tf" -not -path "./**/.terraform/*" -not -path "./test/fixtures/shared/*" -exec bash -c 'terraform validate --check-variables=false $(dirname "{}")' \;
+  find_files . -name "*.tf" -print0 \
+    | compat_xargs -0 -n1 dirname \
+    | sort -u \
+    | compat_xargs -t -n1 terraform validate --check-variables=false
```
Comment (morgante) [inline test/make.sh]: Missing ` -not -path "./test/fixtures/shared/*`

File: Makefile (hunk 1 above)
Comment (aaron-lane) [reply Makefile]: The comments were removed from the template. We will need to address this issue there.

File: Makefile [hunk 4]
```diff
@@ -100,91 +104,94 @@ release-new-version:
...
 	docker run --rm -it \
 		-e BILLING_ACCOUNT_ID  \
-		-e SERVICE_ACCOUNT_JSON \
 		-e DOMAIN \
 		-e FOLDER_ID \
 		-e GROUP_NAME \
 		-e ADMIN_ACCOUNT_EMAIL \
 		-e ORG_ID \
 		-e PROJECT_ID \
+		-e SERVICE_ACCOUNT_JSON \
```
Comment (morgante) [inline Makefile]: As discussed, we should drop this requirement.

**Pull request merged**